app/models/team.rb

```diff
@@ -495,7 +495,11 @@ def filtered_explainers(filters = {})
     query = query.where(updated_at: Range.new(*format_times_search_range_filter(JSON.parse(filters[:updated_at]), nil))) unless filters[:updated_at].blank?
 
     # Filter by text
-    query = query.where('(title ILIKE ? OR url ILIKE ? OR description ILIKE ?)', *["%#{filters[:text]}%"]*3) if filters[:text].to_s.size > 2
+    if filters[:text].to_s.size > 2
+      tsquery = Team.sanitize_sql_array(["websearch_to_tsquery(?)", filters[:text].split(/\s+/).map(&:strip).join(' & ')]) # FIXME: May not work for all languages
```

Comment:
My reading of the `websearch_to_tsquery` documentation is that we can simply use:
`tsquery = Team.sanitize_sql_array(["websearch_to_tsquery(?)", filters[:text]])`
and do not need to split, strip, and join. I've tried the unit test with this alternative and it passes. I'm building the full app to try it end-to-end.